patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -1,34 +1,43 @@
 --- ../src-base/minecraft/net/minecraft/entity/player/EntityPlayer.java
 +++ ../src-work/minecraft/net/minecraft/entity/player/EntityPlayer.java
-@@ -96,6 +96,11 @@
+@@ -96,6 +96,12 @@
  @SuppressWarnings("incomplete-switch")
  public abstract class EntityPlayer extends EntityLivingBase
  {
 +    public static final String PERSISTED_NBT_TAG = "PlayerPersisted";
 +    protected java.util.HashMap<Integer, BlockPos> spawnChunkMap = new java.util.HashMap<Integer, BlockPos>();
 +    protected java.util.HashMap<Integer, Boolean> spawnForcedMap = new java.util.HashMap<Integer, Boolean>();
 +    public float eyeHeight = this.getDefaultEyeHeight();
++    public static final net.minecraft.entity.ai.attributes.IAttribute REACH_DISTANCE = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "generic.reachDistance", 5.0D, 0.0D, 1024.0D).func_111112_a(true);
```

Comment:
Should this be in SharedMonsterAttributes with all the others? Are there any examples of attributes not being in SMA?
Is there any argument for other (non-player) entities using this attribute (i.e. zombies attacking from further away or custom mobs being able to reach to chop the highest branches of trees to chop them)?